posts/2023-09-29-open-liberty-on-aws-ecs-fargate.adoc

```diff
@@ -0,0 +1,684 @@
+
+Example Open Liberty Amazon ECS Task Definition:
+
+[source]
+----
+{
+    "family": "ol-getting-started-blog",
+    "containerDefinitions": [
+        {
+            "name": "open-liberty-getting-started",
+            "image": "icr.io/appcafe/open-liberty/samples/getting-started",
+            "cpu": 512,
+            "memory": 1024,
+            "portMappings": [
+                {
+                    "name": "liberty-getting-started-9080-tcp",
+                    "containerPort": 9080,
+                    "hostPort": 9080,
+                    "protocol": "tcp",
+                    "appProtocol": "http"
+                },
+                {
+                    "name": "liberty-getting-started-9443-tcp",
+                    "containerPort": 9443,
+                    "hostPort": 9443,
+                    "protocol": "tcp",
+                    "appProtocol": "http"
+                }
+            ],
+            "essential": true,
+            "environment": [],
+            "environmentFiles": [],
+            "mountPoints": [],
+            "volumesFrom": [],
+            "logConfiguration": {
+                "logDriver": "awslogs",
+                "options": {
+                    "awslogs-create-group": "true",
+                    "awslogs-group": "/ecs/ol-getting-started-demo",
+                    "awslogs-region": "us-east-1",
+                    "awslogs-stream-prefix": "ecs"
+                }
+            }
+        }
+    ],
+    "executionRoleArn": "",
+    "networkMode": "awsvpc",
+    "requiresCompatibilities": [
+        "FARGATE"
+    ],
+    "cpu": "512",
+    "memory": "1024",
+    "runtimePlatform": {
+        "cpuArchitecture": "X86_64",
+        "operatingSystemFamily": "LINUX"
+    }
+}
+----
+
+A full list of task definition parameters can be found https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html[here].
+
+=== Configuring the Network
+While we are using Amazon ECS with AWS Fargate to manage our runtime, networking components come under Amazon EC2 and still require an amount of management.
+
+For this blog, we want to expose our application externally on protocol default ports: `80` for HTTP and `443` for HTTPS, while using non-default protocol ports for our container.
+
+The networking components can be created during the Amazon ECS Service creation steps. However, issues have been encountered when creating everything using the Amazon ECS Service creation wizard. The primary issue is mapping the different port values between the task definition and the exposing load balancer. The wizard contains an assumption that the port that we will publically expose the service by, is the same as the port that the target group will be forwarding to on the running container. As the port mapping cannot be changed once the target group has been created, the networking components will be created first and then referenced by the service later.
```

Comment:
This this configure network section intro could be more concise. 
It’s also not clear what the “issues” are that it’s describing (doesn’t say why that’s a problem - and I think “problems” is better than “issues” when it means something negative).